<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkDataStorageViewerTestControls</class>
 <widget class="QWidget" name="QmitkDataStorageViewerTestControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Data storage viewer test</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="selectionProviderCheckBox">
     <property name="text">
      <string>Set as selection provider</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QListView" name="selectionListView2"/>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="selectionProviderCheckBox2">
     <property name="text">
      <string>Set as selection provider</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QListView" name="selectionListView"/>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkOnlyImages">
     <property name="text">
      <string>Allow only images </string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="checkOptional">
     <property name="text">
      <string>Is Optional</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="selectionProviderCheckBox4">
     <property name="text">
      <string>Set as selection provider</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkOnlyVisible_2">
     <property name="text">
      <string>Only valid nodes</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="checkOnlyImages_2">
     <property name="text">
      <string>Allow only images
(using NodePredicate feature and general invalid message)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="selectionListenerCheckBox">
     <property name="text">
      <string>Set as selection listener</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="selectionListenerCheckBox2">
     <property name="text">
      <string>Set as selection listener</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkOnlyVisible">
     <property name="text">
      <string>Only valid nodes</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="selectionListenerCheckBox3">
     <property name="text">
      <string>Set as selection listner</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="selectionListenerCheckBox4">
     <property name="text">
      <string>Set as selection listner</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="selectionProviderCheckBox3">
     <property name="text">
      <string>Set as selection provider</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="checkOptional_2">
     <property name="text">
      <string>Is Optional</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QmitkSingleNodeSelectionWidget" name="singleSlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QmitkMultiNodeSelectionWidget" name="multiSlot" native="true"/>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="checkOnlyUneven">
     <property name="text">
      <string>Allow only uneven selection count
(using check funcktion feature)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkMultiNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkMultiNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
